src/integrationTest/java/uk/gov/hmcts/darts/task/runner/impl/DetsToArmAutomatedTaskTest.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.darts.task.runner.impl;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.darts.testutils.PostgresIntegrationBase;
+
+class DetsToArmAutomatedTaskTest extends PostgresIntegrationBase {
```

Comment:
I don't think we need this integration test as it is not verifying anything (can be added when the core logic gets added to the automation task)